app.js

```diff
@@ -0,0 +1,22 @@
+//testing the package I published
+const matrixUtils = require('@definitelynotsam/matrix-utils');
+
+const testMatrix = [
+	[-1, -1],
+	[-1, 0],
+	[-1, 1],
+	[0, 1],
+	[1, 1],
+	[1, 0],
+	[1, -1],
+	[0, -1],
+];
+
+console.log(matrixUtils.loopMatrix(testMatrix));
+
+const newMatrix = matrixUtils.createMatrix(5, 5, () => Math.random() >= 0.6);
+console.log(newMatrix);
+
+const newMatrixClone = matrixUtils.cloneMatrix(newMatrix);
+
+console.log(newMatrix === newMatrixClone);
```

Comment:
I understand this is a file to check if your stuff is working - I'd recommend either not including it in a commit, or writing this using a testing library like `mocha` or `jest`. I like to test run my code on a repl.it since I can iterate somewhat quickly :)